DotBot.py

```diff
@@ -36,6 +39,9 @@ def __init__(self, dotBotId, x, y, floorplan):
         # timestamps of when movement starts/stops
         self.tsMovementStart      = None
         self.tsMovementStop       = None
+        # sequence numbers (to filter out duplicate commands and notifications)
+        self.seqNumMovement       = None
```

Comment:
what does movement mean? Do you refer to commands (Orchestrator->DotBot) and notifications (DotBot->orchestrator)?